packages/components/src/VirtualizedList/ListGrid/ListGrid.test.js

```diff
@@ -68,9 +68,31 @@ describe('ListGrid', () => {
 
 	it('should render no-rows component', () => {
 		// when
-		const wrapper = shallow(<NoRow />);
+		const wrapper = shallow(
+			<ListGrid
+				collection={[]}
+				height={600}
+				id={'my-list'}
+				isSelected={jest.fn()}
+				rowHeight={130}
+				rowRenderer={RowLarge}
+				selectionToggle={jest.fn()}
+				width={1024}
+			>
+				<VirtualizedList.Content
+					label="Id"
+					dataKey="id"
+				/>
+				<VirtualizedList.Content
+					label="Name"
+					dataKey="name"
+				/>
+			</ListGrid>
+		);
+		const NoRows = wrapper.prop('noRowsRenderer');
+		const noRowsWrapper = shallow(<NoRows />);
```

Comment:
I think it will be better to mount instead of shallow, pass empty collection and have the no row final markup in snapshot, instead of extracting the NoRows component. So we test that the List renders the NoRow as wanted